include/seastar/util/backtrace.hh

```diff
@@ -59,13 +61,16 @@ frame decorate(uintptr_t addr) noexcept;
 SEASTAR_MODULE_EXPORT
 template<typename Func>
 void backtrace(Func&& func) noexcept(noexcept(func(frame()))) {
+    #ifdef SEASTAR_HAVE_EXECINFO
```

Comment:
we don't indent `#ifdef` in this project elsewhere. so probably we don't need to do so here?

Reply:
okay changed